packages/x-components/tests/e2e/cucumber/identifier-results/identifier-results.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Identifier results component
+
+  Background:
+    Given an ID results API with a known response
+    And   no special config for identifier results view
+
+  Scenario Outline: 1. ID search with results is made
+    When  "<query>" is searched
+    Then  identifier results are displayed
+
+    Examples:
+      | query |
+      | a02   |
+
+  Scenario Outline: 2. ID search with no results is made
+    When  "<query>" is searched
+    Then  no identifier results are displayed
+
+    Examples:
+      | query |
+      | b123  |
+      | c0 32 |
```

Comment:
I think this should be 2 separate scenarios, one is **a matching identifier is searched but it has no results**, and another one is **a non matching identifier is searched**. In this last case the request shouldn't even be made.